lib/twilio.js

```diff
@@ -0,0 +1,59 @@
+const twilio = require('twilio');
+const Promise = require('Bluebird');
+
+module.exports = {
+  /**
+   * Establishes the connection to Twilio
+   */
+  setupClient() {
```

Comment:
What do you think about renaming this to `getTwilioClient()` and get or create the client. Then in the `verify` and `requestVerification` functions, we can call `getClient().verify` and `getClient().requestVerification` .

Reply:
I'm ok with renaming it, but we only have to call this function once so I'd rather not do `getClient().verify and getClient().requestVerification`

Reply:
What do you think about just instantiating a client directly in this file? like: 
```js
const client = ....

module.exports = {
  ...
};
```